src/datascience-ui/ipywidgets/kernel/index.ts

```diff
@@ -11,6 +11,26 @@ import { SharedMessages } from '../../../client/datascience/messages';
 import { PostOffice } from '../../react-common/postOffice';
 import { WidgetManager } from '../common/manager';
 import { ScriptManager } from '../common/scriptManager';
+
+// Copy of vscode-notebook-renderer old types as of 1.48
+// Keep these so we can support both the old interface and the new interface
+// Interface change here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51675/files
+interface OldNotebookCellOutputMetadata {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    custom?: { [key: string]: any };
+}
+interface OldNotebookOutput {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    data: { [mimeType: string]: any };
+    metadata?: OldNotebookCellOutputMetadata;
+}
+interface OldNotebookOutputEventParams {
```

Comment:
@IanMatthewHuff I think we can remove this code and there's no need for backwards compatiblity.
The only reason i wanted the backwards compatiblity for renderers, is so that it works with point releases of jupyter extension (point release is for current stable).

Reply:
@DonJayamanne Ahh, that's a good call. Wasn't thinking when I moved that code over. You are right, this will only release with matching stable builds. I'll remove that quick.